frames.c

```diff
@@ -28,8 +24,10 @@ void frames_header(uint8_t* frame, uint8_t* header) {
     header[1] = frame[1];
 }
 
-void frames_data(uint8_t frame, uint8_t frame_len, uint8_t* data) {
-    // TODO: implement
+void frames_data(uint8_t* frame, uint8_t frame_len, uint8_t* data) {
+    for (uint8_t i = 4; i < frame_len; i++) {
```

Comment:
I think that there is a bug. It counts from 4 to frame_len - 1, so in example:
frame: AT\x04+DATA#C
with frame_len=10, `i` will be: 4, 5, 6, 7, 8, 9 - so 6 repeats.
Also, I think that memcpy may be faster.
`memcpy(frame + 4, data, frame_len - 6)`
But I didn't tested it yet.

Reply:
Thanks for the catch, I also didn't test this. If you feel like fixing this then go ahead:)